content/en/docs/reference/glossary/kube-scheduler.md

```diff
@@ -10,14 +10,52 @@ aka:
 tags:
 - architecture
 ---
-Control plane component that watches for newly created
-{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
-{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
-to run on.
+A control plane component that watches 
+for new unassigned {{< glossary_tooltip term_id="pod" text="Pods" >}} 
+and chooses a {{< glossary_tooltip term_id="node" text="node">}} for them.
 
 <!--more-->
 
 Factors taken into account for scheduling decisions include:
-individual and collective resource requirements, hardware/software/policy
-constraints, affinity and anti-affinity specifications, data locality,
-inter-workload interference, and deadlines.
+
+- individual and collective resource requirements
+- hardware/software/policy constraints
+- affinity and anti-affinity specifications
+- data locality
+- inter-workload interference
+- deadlines
+
+Individual and collective resource requirements: The CPU and memory requirements 
```

Comment:
@KhushPatibandha I like the bulleted list, but I think the rest might be too much information for the glossary section. I'd suggest we add a link to a page where we talked about these in more detail instead. Maybe this page, https://kubernetes.io/docs/concepts/scheduling-eviction/kube-scheduler/ 

And if we dont have this information written out in the docs yet, and you think it should still be added, please consider creating a new PR adding the content to kube-scheduler page

Reply:
@KhushPatibandha, I'm sorry I lost the thread on this, but agree with @a-mccarthy's feedback here.